Sources/KlaviyoForms/KlaviyoWebView/KlaviyoWebViewController.swift

```diff
@@ -35,6 +36,7 @@ private func createDefaultWebView() -> WKWebView {
 class KlaviyoWebViewController: UIViewController, WKUIDelegate, KlaviyoWebViewDelegate {
     private lazy var webView: WKWebView = {
         let webView = createWebView()
+        webView.customUserAgent = NetworkSession.defaultUserAgent
```

Comment:
for other reviewers: i suggested this since we did it in android too, i figured it might be helpful in nginx logs / splunk to differentiate from standard web